src/modules/savesettings/savesettings-directive.js

```diff
@@ -53,7 +53,7 @@ angular.module('anol.savesettings')
                         // load project name to overwrite
                         if (angular.isUndefined(name) || scope.id) {
                             angular.forEach(scope.projectSettings, function (value) {
-                                if (value.id === scope.id) {
+                                if (value.id === parseInt(scope.id)) {
```

Comment:
When parsing to numeric values, it's best to always provide the base as well.

```suggestion
                                if (value.id === parseInt(scope.id, 10)) {
```